sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileTable.scala

```diff
@@ -34,13 +35,21 @@ import org.apache.spark.sql.util.SchemaUtils
 
 abstract class FileTable(
     sparkSession: SparkSession,
-    options: CaseInsensitiveStringMap,
+    originalOptions: CaseInsensitiveStringMap,
```

Comment:
Do we have chance to use `path` related options in `FileTable`? If not, can we just remove it when create `FileTable`?

Reply:
This seems like a cleaner approach. I updated the PR with this approach.